backend/lcfs/db/models/ProvisionOfTheAct.py

```diff
@@ -0,0 +1,18 @@
+from sqlalchemy import Column, Integer, Float
+from lcfs.db.base import BaseModel, Auditable, DisplayOrder
+from sqlalchemy.orm import relationship
+from sqlalchemy import ForeignKey
+from sqlalchemy import String
+
+
+class ProvisionAct(BaseModel, Auditable, DisplayOrder):
```

Comment:
I think we want to keep the consistency in naming, so this model file name could just be ProvisionAct.py I agree that it makes sense to shorten it to avoid the awkward field names.

Reply:
renamed the file. Thanks for the review.